block/cardano-sl-block.cabal

```diff
@@ -100,8 +100,10 @@ library
                       , ether
                       , exceptions
                       , filepath
+                      , fmt
```

Comment:
I can't say I'm happy with the fact we're using two formatting libraries at the same time.

Reply:
My hope is that we'll switch to `fmt` at some point in the future. I don't want to write any new code that uses `formatting`.